src/databricks/sql/client.py

```diff
@@ -189,6 +193,8 @@ def read(self) -> Optional[OAuthToken]:
         self._session_handle = self.thrift_backend.open_session(
             session_configuration, catalog, schema
         )
+        self.use_cloud_fetch = kwargs.get("use_cloud_fetch", False)
+        self.max_download_threads = kwargs.get("max_download_threads", 10)
```

Comment:
With some performance testing, I did not find too much of a performance increase by allowing more threads than 10. This may vary by individual machine or cluster